dockerfiles/Dockerfile-coda-demo

```diff
@@ -1,42 +1,47 @@
 FROM ubuntu:18.04
```

Comment:
Is this necessary to include in this PR? Would prefer this to be separate (unless we need it for this release for some reason)